### DISCUSSION TITLE
tFileExcelWorkbookOpen

### DISCUSSION THREAD

**Discussion by nadinebenharrath** [jlolling/talendcomp_tFileExcel#39] [General]
Hello, I'm unable to add the component tFileExcelWorkbookOpen in talend v8.0.1 
is it no more compatible in this version ?
thank you in advance

**Comment by jlolling** [comment]
The component runs in about hundreds of projects under Talend 8. can you explain what happens.
First of all, where do you have downloaded the components? From the Release section?

**Comment by jlolling** [comment]
Where do you have downloaded the components? Please download them only from the Release Section (at the right side of the overview page)
